metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/MaeConsumerApplication.java

```diff
@@ -12,13 +13,15 @@
 
 @SuppressWarnings("checkstyle:HideUtilityClassConstructor")
 @SpringBootApplication(exclude = {ElasticsearchRestClientAutoConfiguration.class, CassandraAutoConfiguration.class,
-    SolrHealthContributorAutoConfiguration.class})
-@ComponentScan(excludeFilters = {
-    @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = ScheduledAnalyticsFactory.class)})
+        SolrHealthContributorAutoConfiguration.class})
+@ComponentScan(basePackages = {"com.linkedin.metadata.kafka", "com.linkedin.gms.factory.config",
+    "com.linkedin.gms.factory.common"},
+    excludeFilters = {@ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = ScheduledAnalyticsFactory.class),
+        @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = SiblingGraphServiceFactory.class)}
```

Comment:
not sure why this needs to be excluded as part of this PR

Reply:
They are being injected through other classes which means that when running this in standalone mode you have multiple sources for the same beans.